extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -120,6 +120,7 @@ export let State;
 /** @const @enum {string} */
 export const StateProperty = {
   // Embed options.
+  CAN_ANIMATE_FIRST_PAGE: 'canAnimateFirstPage',
```

Comment:
Nit:
```suggestion
  CAN_RUN_ANIMATIONS_ON_INITIAL_PAGE: ...,
```

Would be a bit more accurate

Reply:
Removing this after our chat.